### PR TITLE
ci: add reactions to /update-vrt comment

### PR DESCRIPTION
**Pull request by UnderKoen**

### CODE REVIEW

File: .github/workflows/update-vrt.yml [hunk 1]
```diff
@@ -4,8 +4,8 @@ on:
     types: [ created ]
 
 permissions:
-  pull-requests: write
-  contents: write
+  pull-requests: read
```
Comment (youngcw) [inline .github/workflows/update-vrt.yml:7]: is this because we don't need write permissions?

Comment (UnderKoen) [reply .github/workflows/update-vrt.yml:7]: From reading the docs, this is not needed for pushing. But I may be wrong

File: .github/workflows/update-vrt.yml [hunk 2]
```diff
@@ -4,11 +4,11 @@ on:
...
+  contents: read
```
[the remark this answers is not in the document]
Comment (youngcw) [reply .github/workflows/update-vrt.yml:8]: @UnderKoen maybe you do need it

[the remark this answers is not in the document]
Comment (UnderKoen) [reply .github/workflows/update-vrt.yml:8]: It seems like it, but I am testing on a private repo to be sure.

[the remark this answers is not in the document]
Comment (UnderKoen) [reply .github/workflows/update-vrt.yml:8]: @youngcw we did need it :( , reworked the workflow to not expose the write secret to any running code from the fork.

[the remark this answers is not in the document]
Comment (UnderKoen) [reply .github/workflows/update-vrt.yml:8]: Thats not true, you only need the permission to push

**Review by matt-fidd — APPROVED** [review]
Looks good

File: .github/workflows/update-vrt.yml [hunk 3]
```diff
@@ -41,9 +41,43 @@ jobs:
...
+        with:
+          repository: ${{ steps.comment-branch.outputs.head_owner }}/${{ steps.comment-branch.outputs.head_repo }}
+          ref: ${{ steps.comment-branch.outputs.head_ref }}
+      - uses: actions/download-artifact@v4
+        continue-on-error: true
+        with:
+          name: patch
+      - name: Apply patch and push
+        run: |
+          git apply Update-VRT.patch
```
Comment (UnderKoen) [inline .github/workflows/update-vrt.yml:80]:
```suggestion
          git apply Update-VRT.patch
          git config --system --add safe.directory "*"
```

**Review by matt-fidd — APPROVED** [review]

**Pull request merged**